src/test/java/com/cloudbees/jenkins/support/filter/SensitiveContentFilterTest.java

```diff
@@ -45,21 +45,23 @@
     @Test
     public void anonymizeSlavesAndLabels() throws Exception {
         SensitiveContentFilter filter = SensitiveContentFilter.get();
-        jenkins.createSlave("foo", "bar", null);
-        jenkins.createSlave("jar", "war", null);
+        // using foo, bar, jar and war could raise flaky test failures. It happened to me when
+        // bar was changed by label_barrier :-O So we use stranger words to avoid this test to be flaky
+        jenkins.createSlave("foostrange", "barstrange", null);
```

Comment:
What does flaky mean here? That the string was not replaced properly?

Reply:
It was replaced correctly, but the new text also contains the word *bar* (label_barrier). So the test failed because the assert said it shouldn't, that's the word replaced. :laughing: random texts

Reply:
Yes, I'm curious as well. Is it just the test that's flaky? Or is the test reflecting flakiness in the behavior?

Reply:
when the test replaces "bar" by "label_aword" the test is successful. But:
when the test replaces "bar" by "label_barrier" the test fails because bar is in the word _again_ :laughing: 

It's a random word, so sometimes it returns the same word or a word with this substring (bar).

Reply:
Ah, so yeah, a flaky test, but not an issue in the implementation.